### PR TITLE
mention about OSCP certification in security.md file and in slides also

### LINKED ISSUE #719: Add mention of certification in Security Roles outline & make a slide.
- [ ] Add a line or 2 of about this certification in our [Security Roles Outline](https://github.com/Techtonica/curriculum/blob/master/roles-in-tech/security.md) based on the following advice. Include a link and a few words about the certification.  
- [ ] Add the same advice in a slide [here](https://docs.google.com/presentation/d/1DjbfFt-rgQzBBPwVSXL1nGww1ziycJIxiSRmWbS-v40/edit#slide=id.p), and be sure to format it like the rest of the slides.

Advice:
One of the security engineers I worked with recommends doing the OSCP certification.  He said it gets you the experience you need to get security jobs.

### PR DESCRIPTION
**Pull request by vsharma20v**
mention about OSCP certification in security.md file and also in security slides and fixed some typo errors in security.md file
Fixed #719

### CODE REVIEW

File: roles-in-tech/security.md [hunk 1]
```diff
@@ -16,27 +16,27 @@
...
+- It would depend on where you want to go (i.e. pen tester, defense, red teaming, compliance, etc.). Someone is considering going into the more technical route but ended up in security compliance which he/she loves so much more. There are so many routes that someone can’t really pin down one but just know that either self-taught, boot camp, or college course, either of one will work. It comes down to if you can show what you know and explain it to interviewers. The info security industry has a huge hiring gap AND they are looking to hire women. You don’t have to know everything but having the attitude that you can learn it and do it says so much more.
```
Comment (Kundan28) [inline roles-in-tech/security.md]: AND -> and or &

Comment (vsharma20v) [reply roles-in-tech/security.md]: @Kundan28  done

File: roles-in-tech/security.md [hunk 2]
```diff
@@ -87,29 +87,29 @@ These are some misconceptions that people have in their mind but it is not alway
...
 
 - **Cyber Security is just a form of defense** - “The best defense is a good offense.”
 
 - **You will never be attacked** - Unfortunately, the attacked business is often the last to know that it's been breached. It's only after banks discover fraudulent charges that businesses hear about a suspected hack.
 
-- **Hackers are criminals** - It is a big misconception. There are type of hackers and not all hackers are criminals. There are three types of hackers called - *White Hat Hackers(Who 'hacks' legally)* , *Grey Hat Hackers(who may sometimes violate laws or typical ethical standards, but do not have the malicious intent)* and *Black Hat Hackers(Illegal hacking)*.
+- **Hackers are criminals** - It is a big misconception. There are the type of hackers and not all hackers are criminals. There are three types of hackers called - *White Hat Hackers(Who 'hacks' legally) *, *Grey Hat Hackers(who may sometimes violate laws or typical ethical standards, but do not have the malicious intent)* and *Black Hat Hackers(Illegal hacking)*.
```
Comment (Kundan28) [inline roles-in-tech/security.md]: `the` is not required and the space before the asterisk(*) is not required.

Comment (vsharma20v) [reply roles-in-tech/security.md]: done

**Pull request merged**